config/config.go

```diff
@@ -15,138 +15,474 @@ package config
 
 import (
 	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
 	"time"
 
-	"github.com/golang/protobuf/proto"
+	"gopkg.in/yaml.v2"
```

Comment:
according to our awesome import grouping rules, this'd actually go in the same group as the `common` one.